src/crypto.js

```diff
@@ -7,24 +7,40 @@ var buf, offset = 0;
 var fcrypto = freedom['core.crypto']();
 // https://developer.mozilla.org/en-US/docs/Web/API/RandomSource/getRandomValues
 var kMaxSingleBuffer = 65536;
+var logMessages = [];
+
+function logMessage(str) {
+  logMessages.push(str);
+}
```

Comment:
semicolons on functions. Does this need to be in both files? could it use `freedom.log` calls?